test/com.cooltt

```diff
@@ -1,20 +1,20 @@
 def mycoe/fun : {
-  (A : (i : dim) -> univ) (B : (i: dim) -> univ)
-  (coe/A : (r : dim) (x : A r) (i : dim) -> sub {A i} {i=r} x)
-  (coe/B : (r : dim) (x : B r) (i : dim) -> sub {B i} {i=r} x)
-  (r : dim) (f : (_ : A r) -> B r) (i : dim) ->
+  (A : (i : 𝕀) -> univ) (B : (i: 𝕀) -> univ)
```

Comment:
The `->` is the only non-Unicode thing on this line!

Reply:
Feel free to add those -> things too.